subprojects/docs/src/docs/release/notes.md

```diff
@@ -34,6 +34,22 @@ The following are the newly deprecated items in this Gradle release. If you have
 ### Example deprecation
 -->
 
+### Local build cache directory cleanup is now time-based
+
+Previously, Gradle would clean up the local build cache directory only if the size of its contents reached 5 GB, or whatever was configured in `targetSizeInMB`.
```

Comment:
I think the release notes entry should have a link to the userguide chapter about the local cache.

Reply:
Good point, added a link.